test/IntegrationTests/docker/azure.Dockerfile

```diff
@@ -0,0 +1 @@
+﻿FROM mcr.microsoft.com/azure-storage/azurite:latest
```

Comment:
Can you pin in it to exact version/label to have more reliable tests?
Dependabot should update this dependency.

Reply:
Updated.